sphinx/source/utils.rst

```diff
@@ -62,6 +62,11 @@ Sampling from GP priors
 .. automodule:: botorch.utils.gp_sampling
 		:members:
 
+Statistical Distances
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. automodule:: botorch.utils.stat_dist
+		:members:
+
```

Comment:
Since the file is now under botorch_community, we may need to revert this. I'll add a page for community docs on the website later in the half.